test/locales.spec.ts

```diff
@@ -12,4 +14,45 @@ describe('locale', () => {
       }
     });
   });
+
+  for (const [localeName, moduleMap] of Object.entries(allLocales)) {
+    describe(localeName, () => {
+      for (const [
+        moduleName,
+        definitionMap,
+      ] of Object.entries<LocaleDefinition>(moduleMap)) {
+        if (moduleName === 'title' || moduleName === 'separator') {
+          continue;
+        }
+
+        describe(moduleName, () => {
+          for (const [definitionName, entries] of Object.entries(
+            definitionMap
+          )) {
+            describe(definitionName, () => {
+              if (Array.isArray(entries)) {
+                it('should have no duplicate entries', () => {
```

Comment:
```suggestion
                it('should not have duplicate entries', () => {
```
Not sure about this one.